### ISSUE TITLE
Fail in the finetuning 

### ISSUE THREAD

**Issue by zhubro321** [deepmodeling/deepmd-kit#4098]
### Bug summary

I use the [model ](https://www.aissquare.com/models/detail?pageType=models&name=DPA-2.2.0-v3.0.0b3&id=272)as pre-trained model, and use [input_torch.json](https://github.com/user-attachments/files/16845984/input_torch.json)as input file. Then execute "dp --pt train input_torch.json --finetune ../DPA-2/OpenLAM_2.2.0_27heads_beta3.pt --model-branch Domains_Alloy"
While it fail and the report show like this:
![image](https://github.com/user-attachments/assets/1cc74bfa-ddf9-4e44-bd13-1cf06c683dea)

I created the image `registry.dp.tech/dptech/deepmd-kit:3.0.0b3-cuda12.1` on the Bohrium platform for calculations, and the model was downloaded from AISsquare as the latest DPA-2 model. Given this, there should not be any compatibility issues. How should I address the problem?

### DeePMD-kit Version

DeePMD-kit v3.0.0b3

### Backend and its version

PyTorch v2.0.0.post200-gc263bd43e8e

### How did you download the software?

Others (write below)

### Input Files, Running Commands, Error Log, etc.

[INPUT.zip](https://github.com/user-attachments/files/16846179/INPUT.zip)

### Steps to Reproduce

None

**Comment by njzjz** [comment]
Please follow the example https://github.com/deepmodeling/deepmd-kit/blob/1abb89b57d02485c8d7e12f93e66554e61647218/examples/water/dpa2/input_torch.json#L8-L45

**Comment by zhubro321** [comment]
Thank you for your suggestion @[njzjz](https://github.com/njzjz) . The finetune step has been trained and run successfully, and the final training model has been saved in: model.ckpt.pt.

Then I want to distill the model, refer to the [document](https://bohrium.dp.tech/notebooks/76262686918)
First, refer to the script in the document to use the finetuned model to label on some data, but report an error message like this:
![b058bae546f66b75b71e330a326f40e](https://github.com/user-attachments/assets/78f205e8-cb1a-4c7b-aaca-c4dfe339a1b6)
It seems unable to recognize descriptor "dpa2". Could you please suggest me a proper way?

**Comment by njzjz** [comment]
`deepmd_pt` is deprecated and not maintained anymore. This document may be out-of-date.
